tests/Fixtures/TestBundle/Serializer/Denormalizer/DiscrAbstractDummyDenormalizer.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
This class should not be necessary. IIRC, API Platform is (or should be) able to automatically detect the type to use for normalization.